gen/helperGen.json

```diff
@@ -0,0 +1,2262 @@
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/outbound_transfers"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/outbound_transfers/:outbound_transfer"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/outbound_transfers/:outbound_transfer/cancel"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/test_helpers/treasury/received_credits"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/received_credits"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/received_credits/:id"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/test_helpers/treasury/received_debits"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/received_debits"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/received_debits/:id"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/transaction_entries"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/transaction_entries/:id"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/transactions"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/treasury/transactions/:id"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints"
+        },
+        {
+            "method": "DELETE",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints/:webhook_endpoint"
+        },
+        {
+            "method": "GET",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints/:webhook_endpoint"
+        },
+        {
+            "method": "POST",
+            "prefix": "",
+            "suffix": "",
+            "url": "/v1/webhook_endpoints/:webhook_endpoint"
```

Comment:
Here in the gen can be added:

`"omittableUrlParts": [
        "v1"
      ],`
      
This way the helpers are simplified a little bit.

Reply:
I'm not sure now how it works, but as an alternative, in order not to modify each path, you can try "omittableUrlParts" at the 1st level, as the "endpointName".